include/quic/endpoint.hpp

```diff
@@ -83,28 +85,29 @@ namespace oxen::quic
 
         // creates new outbound connection to remote; emplaces conn/interface pair in outbound map
         template <typename... Opt>
-        std::shared_ptr<connection_interface> connect(const Address& remote, Opt&&... opts)
+        std::shared_ptr<connection_interface> connect(Address& remote, Opt&&... opts)
```

Comment:
```suggestion
        std::shared_ptr<connection_interface> connect(Address remote, Opt&&... opts)
```
so that it can be called with a const value (it means it gets passed by copy, but that's not a big deal).

Reply:
will commit separately :+1: